### PR TITLE
Empty State To Generic Empty State

### PR DESCRIPTION
**Pull request by mayank-devtron**
# Description

Changed Empty State Component to Generic Empty State Component

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/app/details/cicdHistory/History.components.tsx [hunk 1]
```diff
@@ -138,26 +138,23 @@ export const GitChanges = ({
 }
 
 export const EmptyView = ({ imgSrc, title, subTitle, link, linkText }: EmptyViewType) => {
+
+    const EmptyButton = () => {
+        return (
+            <>
```
Comment (shivani170) [inline src/components/app/details/cicdHistory/History.components.tsx:144]: Remove this fragment in line 144

File: src/components/app/details/triggerView/cdMaterial.tsx [hunk 2]
```diff
@@ -1228,23 +1229,12 @@ export class CDMaterial extends Component<CDMaterialProps, CDMaterialState> {
                 />
             )
         }
-
+        const title="No image available"
+        const subTitle=this.props.materialType === MATERIAL_TYPE.rollbackMaterialList
+            ? 'Previously deployed images will be available here for rollback.'
```
Comment (shivani170) [inline src/components/app/details/triggerView/cdMaterial.tsx:1234]: Import these messaging from constantMessaging component

File: src/components/app/details/triggerView/cdMaterial.tsx [hunk 3]
```diff
@@ -1228,23 +1229,12 @@ export class CDMaterial extends Component<CDMaterialProps, CDMaterialState> {
                 />
             )
         }
-
+        const title="No image available"
```
Comment (shivani170) [inline src/components/app/details/triggerView/cdMaterial.tsx:1232]: Import these messaging for title & subtitle from constantMessaging component

File: src/components/app/details/cicdHistory/History.components.tsx [hunk 4]
```diff
@@ -138,26 +138,23 @@ export const GitChanges = ({
 }
 
 export const EmptyView = ({ imgSrc, title, subTitle, link, linkText }: EmptyViewType) => {
+
+    const EmptyButton = () => {
```
Comment (shivani170) [inline src/components/app/details/cicdHistory/History.components.tsx:142]: renderEmptyButton

File: src/components/app/details/cicdHistory/History.components.tsx (hunk 1 above)
Comment (mayank-devtron) [reply src/components/app/details/cicdHistory/History.components.tsx:144]: This is the enclosing parent div.

**Pull request closed without merging**